### DISCUSSION TITLE
Deathlockdrum (GENERAL) [R]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#50] [Card Balance]
Deathlockdrum (GENERAL) [R]
EQUIPMENT, FIREARM
[S] +2, target's [S] -2.

![image](https://user-images.githubusercontent.com/83040170/120903441-689c0d80-c646-11eb-80c9-78589faa94d0.png)

This card will be more interesting with [S] +3, target's [S] -3. 🤡

**Comment by Sol-au-fein** [comment]
Perfect. I agree.
